### PR TITLE
Removed bad words from documentation

### PR DESCRIPTION
**Pull request by vishalkukreja**
*Issue #, if available:*
#328 

*Description of changes:*
Removed words like 
- technically
- so
- just
- very
- simply
- of course 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: examples/advanced/create-react-app/README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ This is what you should see:
 
 <img alt="screenshot of the React application" src="README.media/react-app-screenshot.png" />
 
-If instead you want to build just the tokens you can run `npm run build-dictionary`. This command will generate the files in the `src/style-dictionary-dist` folder, inside the CRA source folder.
+If instead you want to build the tokens you can run `npm run build-dictionary`. This command will generate the files in the `src/style-dictionary-dist` folder, inside the CRA source folder.
```
Comment (chazzmoney) [inline examples/advanced/create-react-app/README.md]: need to modify this to say we are only building tokens and not running the react app to differentiate it from above

File: examples/advanced/custom-formats-with-templates/README.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ Now open the `build.js` script and look at how these custom formats are declared
 Finally, look at the different template files in the `templates` folder and see how they are built to generate custom file formats in output:
 
 * **web-scss.template**: this is a template that uses [Lodash](https://lodash.com/docs/4.17.10#template), and shows how you can create a custom format for Sass ".scss" files. The same approach can be used if you need other custom formats for the web (or other platforms too).
-* **android-xml.template**: this template too uses Lodash, and shows how to create a custom XML format for Android, that can be read as "resource file". This is just one of the many possible formats for Android, so if you need to create one speak with your developers to agree on the format they want.
+* **android-xml.template**: this template too uses Lodash, and shows how to create a custom XML format for Android, that can be read as "resource file". This is one of the many possible formats for Android, if you need to create one speak with your developers to agree on the format they want.
```
Comment (chazzmoney) [inline examples/advanced/custom-formats-with-templates/README.md]: this sounds funny, should be reworded or cleaned up with appropriate punctuation

File: examples/advanced/custom-formats-with-templates/README.md [hunk 3]
```diff
@@ -50,7 +50,7 @@ Now open the `build.js` script and look at how these custom formats are declared
...
 * **android-xml_alt.hbs**: this is an alternative example of custom XML format for Android, that uses [Handlebar](https://handlebarsjs.com) as templating language.
-* **ios-plist.template**: this template too uses Lodash, and shows how to create a custom PLIST format for iOS, that can be read as "resource file". This is just one of the many possible formats for iOS, so if you need to create one speak with your developers to agree on the format they want.
+* **ios-plist.template**: this template too uses Lodash, and shows how to create a custom PLIST format for iOS, that can be read as "resource file". This is one of the many possible formats for iOS, if you need to create one speak with your developers to agree on the format they want.
```
Comment (chazzmoney) [inline examples/advanced/custom-formats-with-templates/README.md]: this sounds funny, should be reworded or cleaned up with appropriate punctuation

File: examples/advanced/referencing_aliasing/README.md [hunk 4]
```diff
@@ -46,7 +46,7 @@ The reference can point to another property in a **different JSON file**. For ex
       ...
 ```
 
-It is also possible to create **chains of references**, where a value references an alias that is, in turn, another alias of a value, and so on. If you open `button/button.json` you can see that the value of the primary color for the button is an alias of `color.primary.value`, that we just saw above is an alias of `color.base.green.value`:
+It is also possible to create **chains of references**, where a value references an alias that is, in turn, another alias of a value, and so on. If you open `button/button.json` you can see that the value of the primary color for the button is an alias of `color.primary.value`, that we saw above is an alias of `color.base.green.value`:
```
Comment (chazzmoney) [inline examples/advanced/referencing_aliasing/README.md]: the end of this sentence is hard to comprehend now, should be reworded

File: examples/advanced/custom-formats-with-templates/README.md (hunk 2 above)
Comment (vishalkukreja) [reply examples/advanced/custom-formats-with-templates/README.md]: @chazzmoney : Kindly confirm on below statement, added punctuation
This is one of the many possible formats for Android, if you need to create one, speak with your developers to agree on the format they want.

File: examples/advanced/custom-formats-with-templates/README.md (hunk 3 above)
Comment (vishalkukreja) [reply examples/advanced/custom-formats-with-templates/README.md]: @chazzmoney  : added punctuation, same as above
This is one of the many possible formats for iOS, if you need to create one, speak with your developers to agree on the format they want.

File: examples/advanced/referencing_aliasing/README.md (hunk 4 above)
Comment (vishalkukreja) [reply examples/advanced/referencing_aliasing/README.md]: @chazzmoney : please suggest from below options:
1) If you open `button/button.json` you can see that the value of the primary color for the button is an alias of `color.primary.value`, one that we saw above, is an alias of `color.base.green.value`:
2) If you open `button/button.json` you can see that the value of the primary color for the button is an alias of `color.primary.value`, and which we saw above, is an alias of `color.base.green.value`:

**Comment by chazzmoney** [comment]
Hey, just a quick update - we have several of these and are waiting on a larger one to be merged before returning to this to review. We will get back to this soon.

Thanks again for your contribution!

**Comment by dbanksdesign** [comment]
We just merged a large PR with documentation edits. @vishalkukreja could you rebase and fix any conflicts please. Thank you!

**Comment by vishalkukreja** [comment]
> We just merged a large PR with documentation edits. @vishalkukreja could you rebase and fix any conflicts please. Thank you!

@dbanksdesign resolved conflicts.

**Comment by dbanksdesign** [comment]
Closing this as there are no longer relevant changes after the rebase. Thank you for your work on this though and feel free to suggest any other improvements to the docs or the framework!

**Pull request closed without merging**